qa/tasks/cephfs/fuse_mount.py

```diff
@@ -81,10 +81,14 @@ def _mount(self, mntopts, check_status):
             'ceph-fuse', "-f",
             "--admin-socket", "/var/run/ceph/$cluster-$name.$pid.asok",
         ]
+        fuse_cmd = [self._mount_bin, self.hostfs_mntpt]
```

Comment:
@mchangir This is causing failures such as - https://pulpito.ceph.com/yuriw-2023-07-27_03:42:08-fs-wip-yuri10-testing-2023-07-26-0858-pacific-distro-default-smithi/7354247

PTAL